fennel/datasets/datasets.py

```diff
@@ -614,24 +625,14 @@ def aggregate(
 
         # Handle window
         if self.window_field is not None:
-            if len(args) != 0 or len(kwargs) != 0:
-                raise ValueError(
-                    "Aggregations on Tumbling/Hopping/Session window are not yet implemented."
-                )
             if along is not None:
                 raise ValueError(
-                    "Along parameter is not supported with Tumbling/Hopping/Session window."
+                    "'along' param can only be used with discrete windows (Continuous/Forever) and lazy emit strategy."
```

Comment:
discrete windows (Continuous/Forever), this sentence is not adding up for me ?
discrete is Hopping, Tumbling right

Reply:
changed to non-discrete